MAUI/Cartesian-Charts/Axis/Types.md

```diff
@@ -393,17 +408,19 @@ By default, all the series are plotted based on primary and secondary axis.
 
 <chart:SfCartesianChart>
     . . .
+	<chart:SfCartesianChart.XAxes>
+                    <chart:DateTimeAxis Name="series_XAxis" ShowMajorGridLines="False" />
+                </chart:SfCartesianChart.XAxes>
+                <chart:SfCartesianChart.YAxes>
+					<chart:NumericalAxis ShowMajorGridLines="False" />
+                    <chart:NumericalAxis Name="series_YAxis" CrossesAt="{Static x:Double.MaxValue}" ShowMajorGridLines="False" />
     <chart:SfCartesianChart.Series>
         <chart:SfCartesianChart.Series>
             <chart:ColumnSeries ItemsSource="{Binding Data1}" 
                     XBindingPath="Date"
-                    YBindingPath="Value">
-                <chart:ColumnSeries.XAxis>
-                        <chart:DateTimeAxis CrossesAt="{x:Static sys:Double.MaxValue}" ShowMajorGridLines="False"/>
-                </chart:ColumnSeries.XAxis>
-                <chart:ColumnSeries.YAxis>
-                        <chart:NumericalAxis CrossesAt="{x:Static sys:Double.MaxValue}" ShowMajorGridLines="False" />
-                </chart:ColumnSeries.YAxis>
+                    YBindingPath="Value"
+					XAxisName ="series_XAxis"
```

Comment:
@Bhava-Tharani , Remove Name of XAxis, and remove XAxisName ="series_XAxis", as have single axis in XAxes its directly get that axis as default. Do same at C#

Reply:
removed Name of XAxis, and XAxisName ="series_XAxis" from both XAML and C# code snippets